Classes/Notifications/CreateNotificationTitle.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 import StyledTextKit
 
-func CreateNotification(title: String, width: CGFloat, contentSizeCategory: UIContentSizeCategory) -> StyledTextRenderer {
+func createNotification(title: String, width: CGFloat, contentSizeCategory: UIContentSizeCategory) -> StyledTextRenderer {
```

Comment:
Why this change? Not really relevant to the rest of the PR. Could we revert and revisit in another PR if necessary?

Reply:
Ok